libraries/AP_Winch/AP_Winch_Servo.cpp

```diff
@@ -0,0 +1,95 @@
+#include <AP_Winch/AP_Winch_Servo.h>
+
+extern const AP_HAL::HAL& hal;
+
+void AP_Winch_Servo::init(const AP_WheelEncoder* wheel_encoder)
+{
+    _wheel_encoder = wheel_encoder;
+}
+
+void AP_Winch_Servo::update()
+{
+    // return immediately if no servo is assigned to control the winch
+    if (!SRV_Channels::function_assigned(SRV_Channel::k_winch)) {
+        return;
+    }
+
+    if (!_servo_out_init) {
+        // set servo output range
+        SRV_Channels::set_angle(SRV_Channel::k_winch,  1000);
+        _servo_out_init = true;
```

Comment:
moved to init.  I was thinking that it's possible the user might not have set-up the SERVOx_FUNCTION at first and initialising it within the update would allow us to init the range when they do set it up.  Still, it's simpler to do it within the init (saves a variable) and the original method wasn't fool-proof anyway (i.e. if the user moved the channel, the init wouldn't happen).